browser/ui/webui/ai_chat/ai_chat_ui_page_handler.cc

```diff
@@ -67,6 +71,12 @@ AIChatUIPageHandler::AIChatUIPageHandler(
 
   favicon_service_ = FaviconServiceFactory::GetForProfile(
       profile_, ServiceAccessType::EXPLICIT_ACCESS);
+
+  feedback_api_ = std::make_unique<AIChatFeedbackAPI>(
+      owner_web_contents->GetBrowserContext()
```

Comment:
if this is supposed to be anonymous we should not be using the profile url loader factory

Reply:
If it's not ideal to use the profile context `URLLoaderFactory`, then we should change it everywhere we construct `APIRequestHelper` (wallet, news, adaptive captcha, ai chat, and more). Searches in the codebase for those show that we're using the `URLLoaderFactory` from the `BrowserContext` for all those instances. In this instance, like other instances, I believe we're safe because the `APIRequestHelper` does not save cookies, it omits credentials and it disables (and bypasses) cache. 

If it's an issue, then perhaps someone can make a PR which changes them all. As it is, I don't think this warrants an emergency fix or uplift, unless there's a problem with the current implementation that I'm not aware of.